Sources/AudioKitUI/Visualizations/WavetableView.swift

```diff
@@ -0,0 +1,70 @@
+// Copyright AudioKit. All Rights Reserved. Revision History at http://github.com/AudioKit/AudioKitUI/
+
+import AudioKit
+import SwiftUI
+
+class WavetableModel: ObservableObject {
+    @Published var floats: [Float] = Table(.sine).content
+    var node: DynamicOscillator?
+    
+    func updateNode(_ node: DynamicOscillator) {
+        if node !== self.node {
+            self.node = node
+            self.node!.wavetableUpdateHandler = setFloats
+            floats = node.getWavetableValues()
+        }
+    }
+    
+    func setFloats(floats: [Float]) {
```

Comment:
Could use a didSet on floats instead?

Reply:
I don't think so. This is being used to set the ViewModel's floats from the DynamicOscillator's callback. Perhaps it should have a different name to avoid confusion?